proposals/compatibility-mode.md

```diff
@@ -95,6 +95,12 @@ Use of the `sample_mask` or `sample_index` builtins would cause a validation err
 
 **Justification**: OpenGL ES 3.1 does not support `gl_SampleMask`, `gl_SampleMaskIn`, or `gl_SampleID`.
 
+### 8. Disallow two-component (RG) texture formats in storage texture bindings.
+
+Use of the `rg32uint`, `rg32sint`, or `rg32float` texture formats with `STORAGE_BINDING` in a `createTexture()` or `createBindGroupLayout()` call should cause a validation error. Calls to pipeline creation functions with pipeline `layout` set to `auto` and a storage texture binding of those format types in the WGSL shader should cause a validation error.
```

Comment:
Suggestion to more directly say what the spec change would be (I think somewhat better addressing Gregg's comment)

```suggestion
The `rg32uint`, `rg32sint`, and `rg32float` texture formats no longer support the `"write-only" or "read-only" STORAGE_BINDING` capability.

Calls to `createTexture()` or `createBindGroupLayout()` with this combination cause a validation error. Calls to pipeline creation functions with pipeline `layout` set to `"auto"` and a storage texture binding of those format types cause a validation error (in the internal call to `createBindGroupLayout()`).
```

Reply:
> The `rg32uint`, `rg32sint`, and `rg32float` texture formats no longer support the `"write-only" or "read-only" STORAGE_BINDING` capability by default.

Done, but why call out read-only and write-only in particular? This would also affect read-write, no?

Reply:
Mainly because that is the title of the column in https://gpuweb.github.io/gpuweb/#plain-color-formats - these formats already don't have read-write storage capability in core.